modules/local/clonality_classifier.nf

```diff
@@ -0,0 +1,23 @@
+process CLONALITY_CLASSIFIER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "conda-forge::biopython=1.78"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/biopython:1.78' :
+        'biocontainers/biopython:1.78' }"
+
+    input:
+    tuple val(meta), path(indels), path(edition)
```

Comment:
Note that the output of `CIGAR_PARSER` gives these two files in two different channels, `indels` and `edition`. You will have to use the [join](https://www.nextflow.io/docs/latest/operator.html#join) operator to join both channels, this will join them by the meta map, obtaining a channel of the form `[meta, *indels.csv, *subs-perc.csv, *.html, *edits.csv] `, then you will want to sue the [map](https://www.nextflow.io/docs/latest/operator.html#map) operator to return only the elements 0, 1 and 4 of this channel, to get a channel of your required format.